modos/api.py

```diff
@@ -394,23 +387,12 @@ def update_element(
             )
 
         if isinstance(new, model.DataEntity):
-            new_path = Path(new._get("data_path"))
-            new_idx = get_index(new_path)
-            old_path = Path(attr_dict.get("data_path"))
-            old_idx = get_index(old_path)
-
-            if new_path != old_path:
-                self.storage.move(old_path, new_path)
-                if old_idx:
-                    self.storage.move(old_idx, new_idx)
+            new_data = DataElement(new)
+            old_path = attr_dict.get("data_path")
```

Comment:
```suggestion
            old_path = attr_dict.get("data_path")
            # path changed in metadata -> move file inside modos
```

suggestion: high level explanation of why we do this